server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationMemberDao.java

```diff
@@ -55,4 +58,21 @@ public void deleteByOrganizationUuid(DbSession dbSession, String organizationMem
     return mapper(dbSession).selectOrganizationUuidsByUser(userId);
   }
 
+  /**
+   *
+   * @param loginOrganizationConsumer {@link BiConsumer}<String,String> (login, organization uuid)
```

Comment:
Ok, I thought it was automatically generated, I haven't paid attention to the end